src/schema/community-board-budget-request.ts

```diff
@@ -49,3 +62,62 @@ export const cbbrOptionCascade = pgTable(
     ),
   ],
 );
+
+export const cbbrAgencyCategoryResponse = pgTable(
+  "cbbr_agency_category_response",
+  {
+    id: smallint("id").generatedByDefaultAsIdentity().primaryKey(),
+    description: text("description").notNull(),
+  },
+);
+export const communityBoardBudgetRequest = pgTable(
+  "community_board_budget_request",
+  {
+    id: text("id").notNull().primaryKey(),
+    trackingCode: text("tracking_code").notNull(),
+    boroughId: char("borough_id", { length: 1 })
+      .notNull()
+      .references(() => borough.id),
```

Comment:
I think it would be clearer if there were only one reference for the borough_id. I would lean toward the composite definition on line 107 because it's more powerful

 
```suggestion
```